src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+p {
+  margin: 0;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    height: 134px;
+    width: 160px;
+  }
```

Comment:
It's a good practice to use `align-self` instead of `margin: auto` for centering block elements horizontally when they are inside a flex container. Consider using `align-self: center;` on `.card__image` to horizontally center the image.